.prettierignore

```diff
@@ -1,7 +1,4 @@
 .pnp*
 .yarn*
 .vscode*
-packages/examples/src/bakeoff/*/data/*
```

Comment:
what was this used for?

Reply:
This was some thing that I think Hessam was doing to compare different LLMs or something. Not in this repo anyway so we can remove this.